webrender/src/platform/unix/font.rs

```diff
@@ -53,7 +53,56 @@ unsafe impl Send for FontContext {}
 
 extern "C" {
     fn FT_GlyphSlot_Embolden(slot: FT_GlyphSlot);
-    fn FT_GlyphSlot_Oblique(slot: FT_GlyphSlot);
+}
+
+const OBLIQUE_SKEW_FACTOR: f32 = 0.2;
+
+fn get_skew_bounds(bottom: i32, top: i32) -> (f32, f32) {
+    let skew_min = ((bottom as f32 + 0.5) * OBLIQUE_SKEW_FACTOR).floor();
+    let skew_max = ((top as f32 - 0.5) * OBLIQUE_SKEW_FACTOR).ceil();
+    (skew_min, skew_max)
+}
+
+fn skew_bitmap(bitmap: &mut Vec<u8>, width: &mut usize, height: usize, left: &mut i32, top: i32) {
```

Comment:
Passing in a ```&mut usize``` to modify is a bit odd in rust - I think this would be tidier if we return new bounds, or something like that?

Reply:
It seemed a bit obtuse to have to temporarily assign to variables only to then reassign to actual_width/left with those, since tuple assignments are not yet in the language.